kubernetes/salt/config/master.d/salt-master.conf

```diff
@@ -5,11 +5,11 @@ interface: 0.0.0.0
 event_return: mysql
 
 mysql:
-  host: '127.0.0.1'
+  # salt does not support specifying the UNIX socket location here - as a workaround,
```

Comment:
Shouldn't then mysql listen rather on `127.0.0.1` as we had before instead of a unix socket? We wouldn't need the `MYSQL_UNIX_PORT` workaround.